Update `artist_getting_started` to show AYON screenshots (#94)
ynput/ayon-documentation: website/docs/artist_getting_started.md

AYON comes in packages for Windows (10 or Server), Mac OS X (Mojave or higher), and Linux distribution (Centos, Ubuntu).

In AYON you can download the installer from within the website:
![A screenshot of the AYON main menu with the "Download Launcher" menu expanded](assets/ayon_download_installer.png)

A "Download Launcher" blue button should be present, which provides a binary for the Operating System you are accessing the webistes, alternatively you can download a specific binary from the dropdown in the main menu.

> If you are testing or working by yourself, you can also grab installation files from the releases: https://github.com/ynput/ayon-launcher/releases

Reviewer: Opinion on this? I don't like the link explicit like this and ending the sentence without the dot 😅
```suggestion
If you are testing or working by yourself, you can also grab installation files from the [releases](https://github.com/ynput/ayon-launcher/releases).
```

For installation on Windows, download and run the installation file `AYON-#.#.#.exe`.
During the installation process, you can change the destination location path of the application,

> ![A screenshot ofa Windows installation dialog where the person can choose where to install AYON](assets/ayon_install_windows_01.png)

Reviewer: ```suggestion
![A screenshot of a Windows installation dialog where the person can choose where to install AYON](assets/ayon_install_windows_01.png)
```

Drag the OpenPype icon into the Application folder.

> ![A screenshot of the Mac Installation dialog whith the Applications and AYON install logo](assets/ayon_install_macos.png)

Reviewer: ```suggestion
![A screenshot of the Mac Installation dialog with the Applications and AYON install logo](assets/ayon_install_macos.png)
```

:::tip Is AYON running?
AYON runs in the operating system's tray. If you see a green AYON icon in the tray you can easily tell AYON is currently running. Keep in mind that on Windows this icon might be hidden by default, in which case, the artist can simply drag the icon down to the tray.

![A screenshot of the Windows System Tray showing the AYON icon](assets/artist_systray.png)
:::

:::tip Icon not showing on Linux

> Some Linux distributions do not ship with tray icons by defaults, mostly GNOME based, to get the icon, install the following extension: [Appindicator Support for GNOME Shell](https://extensions.gnome.org/extension/615/appindicator-support/)

Reviewer: ```suggestion
Some Linux distributions, mostly GNOME based, do not ship with tray icons by default. To get the icon, install the following extension: [Appindicator Support for GNOME Shell](https://extensions.gnome.org/extension/615/appindicator-support/).
```

:::

## First Launch

When you first start AYON, you will be asked to fill in some basic information.

![A screenshot of the AYON login dialog](assets/artist_login.png)

### AYON Login Details

Your Studio should provide you with the AYON `url`, `username` and `password` to fill in the dialog.

:::tip Example details
URL: `https://ayon.dev/`
username: `johndoe`
password: `v3ry53cur3p455w0rd`
:::

> If you want to try AYON and don't have a server, you can set up an account over at [AYON Cloud](https://ayon.cloud/)

Reviewer: ```suggestion
If you want to try AYON and don't have a server, you can set up an account over at [AYON Cloud](https://ayon.cloud/).
```

## Updates

> AYON updates automatically, based on your studio server, every time you launch it; that why on first time launch it might do a new installation, if your Studio is using a more recent version.on process, even though you might have just installed it.

Reviewer: ```suggestion
AYON updates automatically, based on your studio server, every time you launch it; that's why on first time launch it might do a new installation, if your Studio is using a more recent version, even though you might have just installed it.
```
